umh-core/pkg/communicator/actions/deploy-dataflowcomponent.go

```diff
@@ -551,7 +551,7 @@ func (a *DeployDataflowComponentAction) GetParsedPayload() models.CDFCPayload {
 
 // waitForComponentToBeActive polls live FSM state until the new component
 // becomes active or the timeout hits (→ delete unless ignoreHealthCheck).
```

Comment:
can you explain what string and error in the return parametrers mean? I am a little bit confused as both look like errors